### PR TITLE
fix: October 1st meeting notes

### PR DESCRIPTION
**Pull request by jithine**
## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: meetings/2020-10/oct-01.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Meeting Notes - 2020-10-05
```
Comment (kumada626) [inline meetings/2020-10/oct-01.md]:
```suggestion
# Meeting Notes - 2020-10-01
```

File: meetings/2020-10/oct-01.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Attendees
+
+- [Jithin Emmanuel](https://github.com/jithine)
+- [Tiffany K](https://github.com/tkyi)
+- [Renaud Lepage](https://github.com/renaud-vmg)
+- [Keisuke Kumada](https://github.com/kumada626)
+- [Hiroki Takatsuka](https://github.com/tk3fftk)
```
Comment (kumada626) [inline meetings/2020-10/oct-01.md]: Hiroki was absent this meeting.

**Review by kumada626 — COMMENTED** [review]
small comment.

**Review by kumada626 — APPROVED** [review]

File: meetings/2020-10/oct-01.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#### Work In Progress 
+
+1. Support large build step logs. [#2068](https://github.com/screwdriver-cd/screwdriver/issues/2068)
+1. Build metrics - Display Pipeline downtime information. [#2199](https://github.com/screwdriver-cd/screwdriver/issues/2199)
+1. Build steps are not in order. [#2232](https://github.com/screwdriver-cd/screwdriver/issues/2232)
+   1. Likely an issue with build node. Time in build k8s pod drifted. Have a PR to make sure that UI displays steps in order.
+
+#### Operational issues and feature requests
+1. Limit number of parallel builds in a pipeline. [#2225](https://github.com/screwdriver-cd/screwdriver/issues/2225)
+   1. We had an issue where a single pipeine took down build cluster by running over 100 parallel builds using TURBO resources.
```
Comment (tkyi) [inline meetings/2020-10/oct-01.md]:
```suggestion
   1. We had an issue where a single pipeline took down build cluster by running over 100 parallel builds using TURBO resources.
```

**Pull request merged**